core/src/main/java/io/seata/core/rpc/netty/NettyRemotingServer.java

```diff
@@ -44,6 +47,11 @@
 
     private final AtomicBoolean initialized = new AtomicBoolean(false);
 
+    private ThreadPoolExecutor branchResultMessageExecutor = new ThreadPoolExecutor(NettyServerConfig.getMinBranchResultPoolSize(),
```

Comment:
Why separate it instead of make the it bigger ?

Reply:
Yes,make it bigger can fix too. But I think the key to the problem is not that the thread pool is not large enough, but because the thread pool will be filled up instantly when the concurrency is large, and the completion of these requests depends on subsequent responses, and when the thread pool is full, there is no new threads to process these responses. So, the problem is that the threads already in the thread pool cannot finish the task until timeout.Therefore, I think it is more efficient to separate it instead of make the it bigger.

Reply:
But Rm and Tm they would send the merge message, and then TC receive the merge message would not use your new executor.